server/src/main/java/io/unitycatalog/server/service/Scim2UserService.java

```diff
@@ -198,6 +203,39 @@ public HttpResponse deleteUser(@Param("id") String id) {
     return HttpResponse.of(HttpStatus.OK);
   }
 
+  @Patch("/{id}")
+  public HttpResponse patchUser(@Param("id") String id, PatchRequest patchRequest) {
```

Comment:
Hmm, I'm guessing this is here, because you started your PR branch off of something other than `main`? it should probably clear up once I merge in the SCIM PR you did.

Reply:
yeah i see one pending PR ,  i can also rebase once you merge

Reply:
You should be able to rebase whenever. The PR is for merging into the v0.2 patch branch.